cmd/ak/cmd/events/event.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // Flags shared by the "create", "dispatch", and "list" subcommands.
 var (
-	filename, integration, connectionToken, eventType string
+	filename, integration, connection, eventType string
```

Comment:
Same comment as in dispatch for the other 2 commands.

Reply:
done